aa_stripe/models.py

```diff
@@ -125,6 +125,52 @@ def create_at_stripe(self):
         self.save()
         return self
 
+    def update_at_stripe(self, stripe_token, set_default):
+        is_default = self.customer.default_card.pk is self.pk
+        set_default = is_default if set_default is None else set_default
+
+        if is_default and not set_default:
+            raise StripeLogicalError()
+
+        stripe.api_key = stripe_settings.API_KEY
+
+        customer = stripe.Customer.retrieve(self.customer.stripe_customer_id)
+
+        if is_default and set_default:
+            customer.source = stripe_token
+            customer.save()
+
+        if not is_default and not set_default:
```

Comment:
why?

Reply:
explained in previous comment, I need to change the default source to be able to update the needed card with setting source attribute